package.json

```diff
@@ -69,7 +69,7 @@
     "@leafygreen-ui/checkbox": "11.0.1",
     "@leafygreen-ui/code": "13.0.1",
     "@leafygreen-ui/confirmation-modal": "2.2.2",
-    "@leafygreen-ui/emotion": "^4.0.0",
+    "@leafygreen-ui/emotion": "4.0.3",
```

Comment:
Seems good to me, though we should double check that this console warning goes away after upgrading all of the LeafyGreen packages. (This warning used to exist in Parsley, but it went away after we updated LG components.)

<img width="581" alt="Screen Shot 2022-11-22 at 5 19 56 PM" src="https://user-images.githubusercontent.com/47064971/203433062-a24d8a17-6785-47e8-bd39-7e647b96f337.png">